src/edu/ucsd/library/xdre/tab/TabularRecord.java

```diff
@@ -494,7 +495,7 @@ private void addSubject( Map<String,String> data, Element e, String header,
             addAttribute( el, "parseType", rdfNS, "Collection" );
             if ( element == null ) { element = type; }
 
-            if ( header.endsWith("scientific name") || header.endsWith("common name") 
+            if ( header.endsWith("scientific name") || header.endsWith("common name") || header.endsWith("anatomy") 
```

Comment:
This current pattern of adding new subjects makes me wonder if there's a refactoring opportunity here.

Would it be possible to extract the list of subject headers, labels (and maybe vocabularly -- dams/mads) into a hashmap that could then be enumerated over and/or interrogated for the addSubject and growing if conditional here?

The reason I bring it up is because inevitably we're going to continue to get new subject creation requests. If the change can be made in a single place, that seems ideal. Do you think that would be worth doing in this instance?